docs/team-process.md

```diff
@@ -32,17 +79,39 @@ We strive to be keep our project as Agile as possible.  This means that every sp
 
 Since we strive to work in pairs and mobs, our stand ups focus more on group goals.  Every morning we meet to talk about our goals for our current sprint and also talk about our story goals for the day.  We setup **parking lot** items to give an opportunity to discuss any important topics that the entire team should be aware of.  We will often give small demos of story / bug progress to the Tax Court.
 
-### Sprint Planning 
+ - Time of Ceremony: 9:30 AM CST
```

Comment:
This is a really dumb nit of mine, but I'd make this just `CT` since more than half the year it is CDT.
```suggestion
 - Time of Ceremony: 9:30 AM CT
```